changelogs/unreleased/linearizable-read.md

```diff
@@ -0,0 +1,4 @@
+## feature/core
+* There is a new transaction isolation level now: "linearizable". Transactions
```

Comment:
Nit: We usually add an empty line after the changelog header. Also, I think it's better to start a changelog entry with a sentence in Past Simple: "Added a new transaction level 'linearizable'. ..."

Reply:
Hmm, I don't see anything about features in this document. It says to use Past Tense to described changed or fixed behavior. Example from the text: "Added per-commit testing, updated YUM repositories". In your case it'd be natural to write: Introduced a new transaction level "linearaziable". Transaction started with `box.begin{txn_isolation = "linearizable"}` always see the latest data confirmed by the quorum.

It's minor though so I don't insist - up to you.

Reply:
Here:

> Use the present tense to describe new behavior.
Example
Data changes in read-only mode are now forbidden (gh-5231).

I'd say that new transaction isolation level is new behaviour.